src/javascript/Edit/EditContent.action.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {useContentEditorHistory} from '~/ContentEditorHistory';
+import {useNodeInfo} from '@jahia/data-helper';
+import * as PropTypes from 'prop-types';
+import {useSelector} from 'react-redux';
+import {Constants} from '~/ContentEditor.constants';
+
+export const EditContent = ({context, render: Render, loading: Loading}) => {
+    const {redirect} = useContentEditorHistory();
+    const language = useSelector(state => state.language);
+    const res = useNodeInfo({path: context.path, language}, {getDisplayName: true});
+
+    if (res.loading && Loading && !res.node) {
```

Comment:
```suggestion
    if (res.loading && !res.node) {
```
loading is already required by the component in the prop-types, no need to double check.

It's really necessary to test if there is no res.node ?